cheddar/cheddar-context/src/main/java/com/clicktravel/cheddar/request/context/SecurityContext.java

```diff
@@ -52,4 +52,9 @@
      *         against, see {@link #userId}
      */
     Optional<String> agentUserId();
+
+    /**
+     * @return Optional of the app ID in the security context. Used for pioneer apps
```

Comment:
Probably don't need the pioneer app bit in the comment. In theory this would work for any of our services that needs to use the app model over user model.